index.php

```diff
@@ -30,9 +31,9 @@ function ldaplogin($mail, $pass) {
     }
     
     // Search for the user
-    $search=@ldap_search($con, $ldap_bdn, "(mail=$mail)", array('dn', 'mail'));
+    $search=@ldap_search($con, $ldap_bdn, "(mail=".ldap_escape($mail, "", LDAP_ESCAPE_FILTER).")", array('dn', 'mail'));
```

Comment:
I added this reluctantly because I haven't had the chance to test it, however I think it works just fine.